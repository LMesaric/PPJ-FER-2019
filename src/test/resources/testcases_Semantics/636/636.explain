semantic: invalid implicit conversion from `int` to `char` in return
note: 'return x / 0 ;' on line 10
