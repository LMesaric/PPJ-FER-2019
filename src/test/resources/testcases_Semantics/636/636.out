<naredba_skoka> ::= KR_RETURN(10,return) <izraz> TOCKAZAREZ(10,;)
